Autopilot/Inc/airspeed.hpp

```diff
@@ -84,6 +86,26 @@ class dummyairspeed: public airspeed{
 
 };
 
+class MPXV7002DP : public airspeed {
+    public:
+        MPXV7002DP(const MPXV7002DP*) = delete;
+        static MPXV7002DP* GetInstance();
+        void GetResult(airspeedData_t &Data);
+        void Begin_Measuring();
+        void Init();
+    private:
+        MPXV7002DP();
+        static MPXV7002DP* s_Instance; //single instance of airspeed
+        //command to interface with MPXV7002
+        double readFromMPXV7002DP();
+        int offset = 0;
+        bool dataNew = false;
+        double airspeed = 0; 
+        int sensorStatus = 0; //Im not 100% sure how to check for errors in the device :(
```

Comment:
It's fully analog, so we don't really have any way to do that. Only thing you can do is on adc startup, get the error code from the hal function, if something's wrong, you can set status, otherwise, don't worry about it.